packages/react-components/components/CallToActionsBar.tsx

```diff
@@ -0,0 +1,41 @@
+import TextButton from '@celo/react-components/components/TextButton.v2'
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+
+export interface CallToAction {
+  onPress: () => unknown
```

Comment:
when would this have a return type besides void?

Reply:
I copied this from the existing code we had, I think if this was used with an `async` handler, `void` wouldn't work.